benchmarking/suite.lisp

```
@@ -15,7 +15,7 @@
         ;; the the compilation.
         :for quil-file := (read-quil-file quil-file-path)
         :do
-           (handler-bind ((simple-error
+           (handler-bind ((error
```

Not sure if this change is desirable. This will cause any `error` to be silently swallowed, since `break-on-error` is `nil` when the benchmarks are generated. For example, the error in #346 would probably have gone unnoticed with this change. On the other hand, I'm not sure why this was previously restricted to `simple-error`.

---

🤷‍♂

---

This did make me think "we should have some generic `QUILC-ERROR` type that is a base for all errors raised in quilc."  Maybe I will open an issue and label it `[good first issue]`.

Do you think this should remain `simple-error`?

---

I don't have a stance about this right now. This can change or not change, and either way it should get mentioned in the `[good first issue]` as something to revisit.

---

Reverted for now. Will make sure to link this thread in the new issue.

---

https://github.com/rigetti/quilc/issues/550